src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VarselRepository.kt

```diff
@@ -71,7 +71,16 @@ class VarselRepository(private val database: DatabaseInterface) : IVarselReposit
                 FROM varsel v
                 INNER JOIN vurdering vu
                 ON v.vurdering_id = vu.id
-                WHERE svarfrist <= NOW() AND v.published_at IS NOT NULL AND svarfrist_expired_published_at IS NULL
+                WHERE svarfrist <= NOW() 
+                    AND v.published_at IS NOT NULL 
+                    AND svarfrist_expired_published_at IS NULL
+                    AND NOT EXISTS (
+                        SELECT 1
+                        FROM vurdering vu2
+                        WHERE vu2.personident = vu.personident
+                            AND vu2.created_at > v.created_at
+                            AND vu2.type IN ('OPPFYLT')
```

Comment:
Er det noe forskjell på `type IN ()` og `type == 'OPPFYLT'`?

Reply:
Hm, `IN` gjør at man kan legge opp et sett med verdier. Vi har jo bare foreløpig `OPPFYLT`. Kan godt endre til `==`

Reply:
Jepp, lurte bare på om det var sånn fordi vi har flere verdier i den spørringen i `isaktivitetskrav` og det var copy+paste, eller om det er noen performance-gains eller noe for `IN` 👍🏼

Reply:
Copy+paste. Ikke noe performance gains som jeg vet om

Reply:
Spiller nok ikke noen rolle om man bruker det ene eller det andre her.